src/typings.ts

```diff
@@ -17,6 +18,8 @@ export interface ConfigsOptions extends PrivateConfigsOptions {
   showRememberMe?: boolean;
   isNewTab?: boolean;
   forceLogout?: boolean;
+  authnMethod?: AuthN | AuthN[];
+  samlSubjectId?: string;
```

Comment:
This wrong, this is support to be the same level as `state` or `redirect_uri` etc.